core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/message.groovy

```diff
@@ -13,14 +13,18 @@ dl {
             input(name: "clear", type: "submit", value: _("Dismiss"), class: "submit-button primary")
         }
 
-        text(_("blurb"))
+        p {
+            text(_("blurb"))
+        }
 
-        table(class: "pane sortable bigtable", width: "100%") {
-            tr {
-                th(_("Trigger"))
-                th(_("Most Recent Occurrence"))
-                th(_("Most Recently Occurring Job"))
-                th(_("Duration"))
+        table(class: "pane sortable jenkins-table", width: "100%") {
```

Comment:
```suggestion
        table(class: "sortable jenkins-table", width: "100%") {
```
To remove the sharp borders from the table corners, we can drop the `pane` argument.

Reply:
Updated. I updated the screen above, unfortunately, I had difficulty re-creating an actual slow trigger so it says 0ms.

Reply:
you should be able to update the admin monitor to always be enabled and then return some hardcoded data to trigger the issue